fuzzers/centipede/fuzzer.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration code for centipede fuzzer."""
+
+import subprocess
+import os
+
+from fuzzers import utils
+
+
+def build():
+    """Build benchmark."""
+    # TODO(Dongge): Check if they support trace-dataflow.
+    # TODO(Dongge): Build targets with sanitizers.
+    cflags = ['-fsanitize-coverage=trace-pc-guard,pc-table,trace-cmp']
+    utils.append_flags('CFLAGS', cflags)
+    utils.append_flags('CXXFLAGS', cflags)
+
+    os.environ['CC'] = 'clang'
+    os.environ['CXX'] = 'clang++'
+    os.environ['FUZZER_LIB'] = (
+        '/src/centipede/bazel-bin/libfuzz_target_runner.a '
```

Comment:
This is a bit brittle -- if some refactoring occurs and things are added/removed, then the build may break. Can you add a TODO to fix this? 

We may need to request this from the Centipede team.

Reply:
Yep, this also caused build failure in other benchmarks (e.g. `bloaty_fuzz_target`).